scripts/Diffraction/isis_powder/osiris.py

```diff
@@ -73,23 +88,147 @@ def _load_raw_runs(self, run_details):
         empty_ws_list = osiris_algs.load_raw(empty_numbers, self._drange_sets, "empty", self, run_details.file_extension)
         return sample_ws_list, vanadium_ws_list, empty_ws_list
 
+    def create_vanadiums(self, **kwargs):
```

Comment:
Please could you remove the final "s" so the name matches the function defined on the other instruments? Maybe you've deliberately added the "s" because for Osiris we produce V files for more than one dRange but I think on balance it's better to align the function names with the other instruments. Then the reduction script for all instruments is the same ie call `create_vanadium` followed by `focus`.